dataeng/jobs/createJobsNew.groovy

```diff
@@ -52,7 +52,6 @@ for (task in taskMap) {
     catch (Exception e) {
         out.println("File $extraVarsFileName does not exist.")
     }
-
```

Comment:
If changes in this code were reverted, try not to make the diff show that it was touched.